src/effects/Mask.cpp

```diff
@@ -110,15 +110,12 @@ std::shared_ptr<openshot::Frame> Mask::GetFrame(std::shared_ptr<openshot::Frame>
 		// Get the average luminosity
 		int gray_value = qGray(R, G, B);
 
-		// Adjust the contrast
-		float factor = (259 * (contrast_value + 255)) / (255 * (259 - contrast_value));
-		gray_value = constrain((factor * (gray_value - 128)) + 128);
-
 		// Adjust the brightness
 		gray_value += (255 * brightness_value);
 
-		// Constrain the value from 0 to 255
-		gray_value = constrain(gray_value);
+		// Adjust the contrast
+		float factor = (20 / std::fmax(0.00001, 20.0 - contrast_value));
+		gray_value = (factor * (gray_value - 128) + 128);
```

Comment:
I see the constrain removed and the factor seems to be simpler.  Looks okay though